_pytest/config.py

```diff
@@ -399,13 +399,15 @@ def consider_conftest(self, conftestmodule):
             self.consider_module(conftestmodule)
 
     def consider_env(self):
-        self._import_plugin_specs(os.environ.get("PYTEST_PLUGINS"))
+        specs = os.environ.get("PYTEST_PLUGINS")
+        if specs:
+            plugins = specs.split(',')
```

Comment:
Thanks for the reply @RonnyPfannschmidt 

> 1. we now pass more different types trough the functions (before it was None or a string, now it is None, a string or a list)

You mean the change in `consider_env`? I don't see a behavior change there, because it went from passing `None` or `str` to passing a `list[str]` to `_import_plugin_specs`, which was always supported by `_import_plugin_specs` anyway. Or do you mean that previously `consider_env` would always call `_import_plugin_specs`, even with `None`? I guess we can change `consider_env` so that it calls `_import_plugin_specs` even when `PYTEST_PLUGINS` is not defined, but it seems to me an stretch to consider this a "behavior change" (which I'm assuming you mean to put this fix into `features` instead of `master`). If that's what you mean, could you provide an example of a situation where this would affect users at all, considering that the `config` module is rarely (if ever) overwritten by plugins and users?